shavar/templates/swagger.mako

```diff
@@ -0,0 +1,100 @@
+swagger: "2.0"
+host: ${HOST}
+info:
+  title: Shavar Service
+  description: Mozilla's implementation of the safe browsing protocol
+  contact:
+    name: Ryan Tilder
+    email: rtilder@mozilla.com
+  license:
+    name: MPL v2.0
+    url: https://www.mozilla.org/en-US/MPL/2.0/
+  version: ${VERSION}
+  x-mozilla-services:
+    homepage: https://github.com/mozilla-services/shavar
+schemes:
+    - ${SCHEME}
+paths:
+    /__heartbeat__:
+      get:
+        operationId: getHeartbeat
+        description: Performs an heartbeat on the app
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The hearbeat was successfull
+    /__version__:
+      get:
+        operationId: getVersion
+        description: Returns the version
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The version is returned
+    /list:
+      get:
+        operationId: getList
+        description: Returns the lists names
+        produces:
+        - text/plain
+        responses:
+          '200':
+            description: The lists names are returned
+      post:
+        operationId: postList
+        description: Returns the lists names (dupe of GET)
+        produces:
+        - text/plain
+        responses:
+          '200':
+            description: The list names are returned
+    /gethash:
+      post:
+        operationId: getHash
+        description: Returns the data in chunks per line {list_name}:{chunk_number}:{data_len}\n{data}
+        produces:
+        - application/octet-stream
```

Comment:
It produces `application/octet-stream` on 200, `text/html` on 400. Do we need to enumerate that for swagger?